src/main/java/com/iemr/hwc/service/registrar/RegistrarServiceImpl.java

```diff
@@ -746,6 +744,39 @@ public String registerBeneficiary(String comingRequest, String Authorization) th
 		return response1.toString();
 	}
 
+	// beneficiary advance search new integrated with common and identity
+	public String getBeneficiaryByBlockIDAndLastModDate(String blockID, Date lastModifDate, String Authorization) {
+		String returnOBJ = null;
+		try {
+			RestTemplate restTemplate = new RestTemplate();
+			JSONObject obj = new JSONObject();
+			obj.put("blockID", Integer.parseInt(blockID));
+			obj.put("lastModifDate", lastModifDate.getTime());
+
+			if ((obj.has("blockID") && !obj.isNull("blockID"))){
```

Comment:
What are this empty if conditions?

Reply:
It was just a test when I was debugging. I can remove it

Reply:
Change done. You can review again please @roopesh-beehyv